quickstart/README.md

```diff
@@ -9,31 +9,28 @@ You can use it as a baseline to write your own first kusto client application, a
 
 ### Retrieving the app from GitHub
-1. Download the app files from this GitHub repo
-2. Modify `kusto_sample_config.json`, changing `KustoUri`, `IngestUri` and `DatabaseName` appropriately for your ADX cluster
+1. Download the app files from this GitHub repo.
+1. Modify the `kusto_sample_config.json` file, changing `KustoUri`, `IngestUri` and `DatabaseName` appropriately for your cluster.
 
 ### Retrieving the app from OneClick
-1. Open a browser and type your cluster's URL (e.g. https://myadxcluster.westeurope.kusto.windows.net/), and you will be redirected to the _Azure Data Explorer_ website
-2. Open the left-side pane via the hamburger menu, if it isn't already open
-3. On the left-side pane, choose _Data_
-4. Click on _Generate Sample App Code_ button
-5. Follow the wizard
-6. Download the app as a zip file
-7. Extract the app source code to your folder of choice
-8. The configuration parameters defined in `kusto_sample_config.json` are already defined appropriately for your ADX cluster, which you can verify
-
-### Running the app
-1. Open a command line window to the folder extracted to above
-2. Run `mvn clean install` to compile the source code into a binary
-3. Run the binary using `java -jar target\kusto-quickstart-[version]-jar-with-dependencies.jar`
-
-### Optional Changes
-1. Within the app itself, you may alter the default User-Prompt authentication method by editing `authenticationMode`
-2. You can also make the app run without stopping between steps by setting `waitForUser = False`
-
-###Troubleshooting
-* If you are having trouble running the app from your IDE, first check if the app runs from command line, then consult the troubleshooting references of your IDE.
\ No newline at end of file
+1. Open a browser and type your cluster's URL (e.g. https://mycluster.westeurope.kusto.windows.net/). You will be redirected to the _Azure Data Explorer_ Web UI.
+1. On the left menu, select **Data**.
+1. Click **Generate Sample App Code** and then follow the instructions in the wizard.
+1. Download the app as a ZIP file.
+1. Extract the app source code. **Note**: The configuration parameters defined in the `kusto_sample_config.json` file are preconfigured with the appropriate values for your cluster. Verify that these are correct.
+
+### Run the app
+1. Open a command line window and navigate to the folder where you extracted the app.
+1. Run `mvn clean install` to compile the source code into a binary.
+1. Run the binary using `java -jar target\kusto-quickstart-[version]-jar-with-dependencies.jar`.
+
+### Optional changes
+1. Within the app itself, you can change the default User-Prompt authentication method by editing `authenticationMode`
+1. You can also make the app run without stopping between steps by setting `waitForUser = False`
+
+### Troubleshooting
+* If you are having trouble running the app from your IDE, first check if the app runs from the command line, then consult the troubleshooting references of your IDE.
```

Comment:
We haven't talked about IDE until now (only Web UI), where did we start using this?

Reply:
We don't have anything to do with IDEs, but the average user will be using one. Since the intent here is to cater to the most beginner of users who aren't even sure how to set up their environment, we wanted to provide some direction for getting that set up.

Reply:
I would move this to the run section

Reply:
I think that line is a helpful troubleshooting idea to someone who tries and fails to get it running in their IDE. I think a troubleshooting section is quite helpful.

Reply:
Right, but it's out of context as it's placed after the optional settings and away from the run section that it applies to. If you want to retain the heading, which I don't thin is necessary for 1 item, then maybe have it as a subsection of run.

Reply:
Ah, good point. I moved the Troubleshooting section to be a subsection of Run.